src/Interfaces/LazySet.jl

```diff
@@ -251,6 +252,30 @@ Alias for the support vector σ.
 """
 const support_vector = σ
 
+"""
+    isboundedtype(S::LazySet)
```

Comment:
```suggestion
    isboundedtype(::LazySet)
```

Reply:
`S` is referred to in the docstring.

Reply:
With the new implementation this is actually needed.